sys-kernel/gentoo-kernel/gentoo-kernel-6.3.8-r1.ebuild

```diff
@@ -44,7 +44,7 @@ S=${WORKDIR}/${MY_P}
 
```

Comment:
Revbump doesn't seem appropriate here. Nothing should change for people not using modules-sign, should it?